src/frontend/Util_ROM.cpp

```diff
@@ -461,6 +461,37 @@ int LoadROM(const char* file, int slot)
     }
 }
 
+u32* RomIcon(u8 data[512], u16 palette[16])
```

Comment:
if the data size is known to the caller and never too large, it's preferable for the caller to pass in a pointer where the data can be stored so the stack can be used for this.

Reply:
RomIcon? the capitalization there is inconsistent with the rest of the code

Reply:
> 
> 
> RomIcon? the capitalization there is inconsistent with the rest of the code

Should I change it to ROMIcon?